src/pages/data-management/components/FileUploadModal.jsx

```diff
@@ -0,0 +1,210 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Modal,
+  Button,
+  Typography,
+  Select,
+  Space,
+  Row,
+  Col,
+  Empty,
+  Divider,
+  List,
+} from 'antd';
+import { CheckCircleTwoTone, CloseCircleTwoTone } from '@ant-design/icons';
+import Dropzone from 'react-dropzone';
+
+const { Text, Title, Paragraph } = Typography;
+const { Option } = Select;
+
+const NewProjectModal = (props) => {
+  const { visible, onUpload, onCancel } = props;
+
+  const initialSelected = '10X Chromium';
+  const [selectedTech, useSelectedTech] = useState(initialSelected);
+  const [canUpload, setCanUpload] = useState(false);
+  const [filesList, setFilesList] = useState([]);
+
+  const techOptions = {
+    '10X Chromium': {
+      acceptedFiles: [
+        'barcodes.tsv',
+        'barcodes.tsv.gz',
+        'features.tsv',
+        'features.tsv.gz',
+        'matrix.mtx',
+        'matrix.mtx.gz',
+      ],
+      validMimeTypes: ['text/tsv', 'application/gzip'],
+      inputInfo: [
+        'barcodes.tsv or barcodes.tsv.gz',
+        'features.tsv or features.tsv.gz',
+        'matrix.mtx or matrix.mtx.gz',
+      ],
+    },
+  };
+
+  useEffect(() => {
+    setCanUpload(filesList.length);
+  }, [filesList]);
+
+  // Handle on Drop
+  const onDrop = (acceptedFiles) => {
+    const newList = [];
+
+    const acceptedFilesRegexp = `(${techOptions[selectedTech].acceptedFiles.join('|')})$`;
```

Comment:
Enforce file naming convention here., as decribed under `techOptons[techName].acceptedFiles`, and informed to the user through `inputInfo`